src/Components/BackupDestinationListRecords.php

```diff
@@ -47,7 +48,34 @@ public function render(): View
     public function table(Table $table): Table
     {
         return $table
-            ->query(BackupDestination::query())
+            ->records(
+                function (?string $sortColumn, ?string $sortDirection, ?string $search) {
+                    $data = [];
+
+                    foreach (FilamentSpatieLaravelBackup::getDisks() as $disk) {
+                        $data = array_merge($data, FilamentSpatieLaravelBackup::getBackupDestinationData($disk));
+                    }
+
+                    return collect($data)
+                        ->when(
+                            filled($sortColumn),
+                            fn (Collection $data): Collection => $data->sortBy(
+                                $sortColumn,
+                                SORT_REGULAR,
```

Comment:
Would SORT_NATURAL not be better as numbers will be sorted correctly?

```suggestion
                                SORT_NATURAL,
```